src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingParameterResolver.cs

```diff
@@ -74,28 +75,36 @@ protected string GetFormatOrNull(string value)
             return null;
         }
 
-        // This is an alias for 'sys.randguid'
-        private class RandGuidResolver : BindingParameterResolver
+        // This is an alias for 'sys.guid'
+        private class GuidResolver : BindingParameterResolver
```

Comment:
>GuidResolver [](start = 22, length = 12)

I'd prefer we deprecate (ie, not expand) this entire path, not add any new BindingParameterResolver, and instead push people down  {sys} instead.